client/signup/steps/woocommerce-install/step-business-info/index.tsx

```diff
@@ -40,6 +45,10 @@ export default function StepBusinessInfo( props: WooCommerceInstallProps ): Reac
 		updateOnboardingProfile( 'selling_venues', venue );
 	}
 
+	function updateRevenue( venue: string ) {
```

Comment:
any reason why the parameter is called venue (?) shouldn't it be revenue? not a big deal tho